assets/js/params/Custom/VersionSelect/DataVersionSelect.js

```diff
@@ -19,27 +25,48 @@ export class DataVersionSelect extends React.PureComponent {
   handleCloseModal = () => this.setState({ isDataVersionModalOpen: false })
 
   render () {
-    const { wfModuleId, currentVersionIndex, nVersions, isReadOnly } = this.props
+    const { wfModuleId, currentVersionIndex, nVersions, isReadOnly, i18n } = this.props
     const { isDataVersionModalOpen } = this.state
 
     let inner
 
     if (nVersions === 0) {
       inner = (
         <>
-          <div className='label'>Version</div>
+          <div className='label'>
+            <Trans id='js.params.Custom.VersionSelect.DataVersionSelect.noVersions.label'>
+                Version
+            </Trans>
+          </div>
           <div className='no-versions'>–</div>
         </>
       )
     } else if (isReadOnly) {
       inner = (
-        <div className='read-only'>Version {nVersions - currentVersionIndex} of {nVersions}</div>
+        <div className='read-only'>
+          <Trans id='js.params.Custom.VersionSelect.DataVersionSelect.readOnly.label'>
+            Version {nVersions - currentVersionIndex} of {nVersions}
+          </Trans>
+        </div>
       )
     } else {
       inner = (
         <>
-          <div className='label'>Version</div>
-          <button type='button' title='Select version' onClick={this.handleClickOpenModal}>{nVersions - currentVersionIndex} of {nVersions}</button>
+          <div className='label'>
+            <Trans id='js.params.Custom.VersionSelect.DataVersionSelect.selectVersion.label'>
+                Version
+            </Trans>
+          </div>
+          <button
+            type='button'
+            title={i18n._(t('js.params.Custom.VersionSelect.DataVersionSelect.selectVersion.hoverText')`Select version`)}
+            onClick={this.handleClickOpenModal}
+          >
+            <Trans id='js.params.Custom.VersionSelect.DataVersionSelect.versionCount' description='The first parameter will be the current version and the second will be the number of versions'>
```

Comment:
The parameter `{1}` is already named `{nVersions}` in the translation message. I shall modify the comment for clarification.
This string is not actually duplicated, since for some reason in the one case it is all together and in the other case it is split by a button and is in different `<div>`s.